modules/common/src/main/java/com/opengamma/sdk/common/ServiceInvoker.java

```diff
@@ -157,6 +158,33 @@ public Response intercept(Chain chain) throws IOException {
     }
   }
 
+  //An interceptor that adds the User-Agent header and exposes useful information about the SDK and runtime.
+  private class UserAgentHeaderInterceptor implements Interceptor {
+
+    @Override
+    public Response intercept(Chain chain) throws IOException {
+      Properties systemProperties = System.getProperties();
+      String userAgentHeader = "opengamma-sdk-java/" +
```

Comment:
Seems like this will always be the same, so set it up in a static variable (with a catch to set it to a simple constant string if the properties cannot be accessed)